app/views/MakingALateAppealPage.scala.html

```diff
@@ -23,9 +23,15 @@
         characterCount: components.characterCount,
         formHelper: FormWithCSRF)
 
-@(form: Form[_], pageMode: PageMode)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@(form: Form[_], pageMode: PageMode, numberOfLatePenalties: Option[Int] = None)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
```

Comment:
Could you not use the session key? I'm thinking with this, it seems we would call the backend to retrieve the amount of penalties which could mean we are ignoring what the user selected on the 'do you want to appeal both penalties' page (seems far fetched I know but could easily be coded like that)

We could implement this logic now, by:
- getting the session key (doYouWantToAppealBothPenalties) value
- pass a boolean to the view (isAppealingBothPenalties)
- then you can apply similar logic you've done here

Let me know what you think

Reply:
I think that sounds more elegant, though would we need the boolean if we've gotten the amount from the session key?

Reply:
The session key doesn't store the amount of penalties, it stores whether the user clicked yes or no, so if they clicked yes then we can set it to 'The penalties...' and if they selected no, it would be 'the first penalty...'

Reply:
Ohh, so the "first penalties" is when you have the option to appeal two but you don't. That makes sense. I'll change some test descriptions as well.